packages/tools/client-debugger/client-debugger-view/src/components/AudienceHistoryTable.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import {
+	Avatar,
+	TableBody,
+	TableCell,
+	TableRow,
+	Table,
+	TableHeader,
+	TableHeaderCell,
+} from "@fluentui/react-components";
+import { Clock20Regular } from "@fluentui/react-icons";
+import { FilteredAudienceHistoryData } from "./AudienceView";
+
+/**
+ * Input for {@link AudienceHistoryTable}
+ */
+export interface AudienceHistoryTableProps {
+	audienceHistoryItems: FilteredAudienceHistoryData[];
+}
+
+/**
+ * Render audience history in {@link AudienceView}
+ */
+export function AudienceHistoryTable(props: AudienceHistoryTableProps): React.ReactElement {
+	const { audienceHistoryItems } = props;
+
+	// Columns for rendering audience history
+	const audienceHistoryColumns = [
+		{ columnKey: "clientId", label: "ClientId" },
+		{ columnKey: "time", label: "Time" },
+	];
```

Comment:
Nit: maybe we want to add a third column for "Event" with values Join/Leave? For users who are seeing this information for the first time, the color coding might not be enough to fully explain what each row is about. cc @rohandubal ?

Reply:
Interesting. I agree that color could not be enough for users using the devtools for the first time. I would like to have more thoughts on this.

Reply:
Also for accessibility purposes, we can't depend strictly on colors. We should definitely re-add the icons somewhere. I personally like the "event" column idea.

Reply:
And actually, I almost wonder if the coloring is too bright / noisy. Maybe **just** icons would be better?

Reply:
`event` column added to note if the audience was `added` or `removed`

Reply:
Can we move the event column to be the first column? I think that's the most important contextual info we're presenting in that view.